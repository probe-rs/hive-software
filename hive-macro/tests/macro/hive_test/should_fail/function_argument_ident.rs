#[hive_macro::hive_test]
fn test_function(wrong_ident: String, handle: String, info: String, define: String) {}

fn main() {}
