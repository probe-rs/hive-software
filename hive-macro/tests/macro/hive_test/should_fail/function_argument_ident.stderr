error: Wrong function argument name

         = help: For better readability Hive testfunctions should have the same argument names as shown in the example below.
         = note: example: fn my_fancy_test(test_channel: &mut dyn TestChannelHandle, session: &mut Session, target_info: &HiveTargetInfo, defines: &DefineRegistry);

 --> tests/macro/hive_test/should_fail/function_argument_ident.rs:2:18
  |
2 | fn test_function(wrong_ident: String, handle: String, info: String, define: String) {}
  |                  ^^^^^^^^^^^
